### ISSUE TITLE
WCAG 2.5.5: Touch inputs must have a sufficient target size. (android.widget.ImageButton)

### ISSUE THREAD

**Issue by petra1** [petra1/AccessibilityDemo#10]
#### Issue
`Touch inputs must have a sufficient target size.` ([TouchSizeWcag](https://accessibilityinsights.io/info-examples/android/touch-size-wcag/))
#### Target application
ch.petraaccess_for_all.accessibilitydemo
#### Element path
android.widget.ImageButton
#### How to fix
    The element has an insufficient target size (width: 47.619dp, height: 43.047dp). Set the element's minWidth and minHeight attributes to at least 44dp.

![grafik](https://user-images.githubusercontent.com/10880024/119220007-84e96780-bae8-11eb-807a-01a9177602bb.png)
